corehq/apps/reports_core/filters.py

```diff
@@ -115,8 +115,8 @@ def date_or_nothing(param):
             return DateSpan(startdate, enddate, inclusive=date_range_inclusive)
 
     def default_value(self):
-        # default to a week's worth of data.
-        return DateSpan.since(7)
+        # default to "Show All Dates"
+        return None
```

Comment:
Should only affect evaluating reports when no date range is provided for a filter.  This affects saved UCRs as in the ticket but does not affect UCRs rendered in the browser.  As far as I know reports_core/filters.py is only used in UCR